test-fixtures/aleph-sequential/yesF001

```diff
@@ -0,0 +1,25 @@
+LDR    
```

Comment:
This record is not in AlephSeq format

Reply:
It is marc-json-js .toString() format. (https://github.com/NatLibFi/marc-record-js/blob/3694d70041463e415fa9bbb473fb441cd2845081/src/index.js#L165-L179)